### ISSUE TITLE
Buffer is not a string

### ISSUE THREAD

**Issue by jimmywarting** [archiverjs/archiver-utils#4]
I keep seeing this in my error log:

```
error Error: If encoding is specified then the first argument must be a string
     at new Buffer (buffer.js:106:13)
     at Readable.<anonymous> (../node_modules/archiver-utils/index.js:39:15)
```

I'm using node v6 i know they made a lot of changes/improvments with buffer

**Comment by jimmywarting** [comment]
Recently saw a pull request for this

**Comment by ctalkington** [comment]
yah #2 most likely, ill review it this weekend and see if we cant get a patch release out.

**Comment by tchoulihan** [comment]
I'm seeing this issue too.

**Comment by h2non** [comment]
I'm experimenting the same issue in node.js `+6`

**Comment by ctalkington** [comment]
A fix has been rolled out as v1.2.0.

**Comment by h2non** [comment]
Great, thank you!
